test/test.py

```diff
@@ -0,0 +1,98 @@
+import munge
+import unittest
+
+
+def file_factory(n):
```

Comment:
When possible, use descriptive naming: e.g. this could be `file_whale_factory` and `file_fact_2` could be `file_angel_factory`.